fabtests/benchmarks/dgram_pingpong.c

```diff
@@ -129,7 +129,7 @@ int main(int argc, char **argv)
 	hints->caps = FI_MSG;
 	hints->mode |= FI_CONTEXT;
 	hints->domain_attr->mr_mode = opts.mr_mode;
-	hints->domain_attr->threading = FI_THREAD_DOMAIN;
+	hints->domain_attr->threading = FI_THREAD_DOMAIN_RELAXED;
```

Comment:
All providers don't support this threading model. Drop the changes or make the new threading model an option.

Reply:
Why don't all providers support the new threading model?

Reply:
Some providers (e.g. opx) doesn't recognize this new value and will give runtime error. Some other providers accept this value but won't perform intended lock optimization. Until all providers have been changed to take act properly on this threading model, let's keep this as opt-in.

Reply:
Agree with j-xiong. This should be an opt-in model

Reply:
@j-xiong a related topic: I find fabtests always use `FI_TC_LOW_LATENCY` in tclass but it's currently ignored by all providers. Should we remove it in fabtests as the default tclass?

Reply:
I searched for FI_THREAD_DOMAIN in the code, and it doesn't seem like providers do specific actions for it.

Reply:
@shijin-aws Yes, I think that makes sense.